client/signup/config/flows.js

```diff
@@ -119,20 +118,9 @@ function filterDesignTypeInFlow( flowName, flow ) {
  * @return {string}          New flow name.
  */
 function filterFlowName( flowName ) {
-	if ( user.get() ) {
-		// logged in, don't allow user-first
-		if ( flowName === 'user-first' ) {
-			flowName = 'main';
-		}
-	} else {
-		// don't allow user-first phase two when logged out, has no user step
-		if ( flowName === 'user-continue' ) {
-			flowName === 'main';
-		}
-		// logged out, in main flow, maybe enter user-first abtest
-		if ( flowName === 'main' && abtest( 'userFirstSignup' ) === 'userFirst' ) {
-			flowName = 'user-first';
-		}
+	// temporary while there are still emails out there that will take people here
```

Comment:
This is unnecessary if we remove the `user-continue` flow. Undefined flows redirect to `main`.

Reply:
The more red, the merrier. Nice thinking.